### PR TITLE
create an api to get data from the database

### PR DESCRIPTION
**Pull request by khaledkzy**

### CODE REVIEW

File: server/dbClients/placesDB.js [hunk 1]
```diff
@@ -8,4 +8,8 @@ const addNewPlace = (query, callback) => {
     Place.create(query).then(callback)
 };
 
-module.exports = addNewPlace;
\ No newline at end of file
+const getPlaces = (query, callback) => {
+    mongoose.connect(connection);
```
Comment (rarmatei) [inline server/dbClients/placesDB.js]: `mongoose.connect(connection);` only needs to be called once. We don't need to call it in every function where we use Mongoose. 

Could we move it into "connection.js" and call it there?

File: server/routes/api.js [hunk 2]
```diff
@@ -1,5 +1,11 @@
 const express = require('express');
 const router = express.Router();
+const connection = require('../dbClients/connection')
+const Place = require('../models/Place');
+const mongoose = require('mongoose')
```
Comment (kabaros) [inline server/routes/api.js]: this mongoose, model stuff should not be here (it's only needed in placesDB - that is the module that has the responsibility of talking to DB)

File: server/routes/api.js [hunk 3]
```diff
@@ -14,4 +20,21 @@ router.get('/questions', function (req, res, next) {
     );
 });
 
+router.get('/places', function (req, res, next) {
```
Comment (kabaros) [inline server/routes/api.js]: you have two routes for /places now - one needs to be out

**Comment by kabaros** [comment]
[API / GET places](https://trello.com/c/a0dvIefW/37-api-get-places)

**Pull request merged**